mysite/templates/mysite/profile.html

```diff
@@ -0,0 +1,11 @@
+{% extends "mysite/base_generic.html" %}
+
+{% block content %}
+
+{% if user.is_authenticated %}
+    <p><strong>Username:</strong> {{ user.username }}</p>
```

Comment:
You can also write these lines as 
<p> {{ user.username.label_tag }} {{ user.username }}</p>

Reply:
Label tag not working on user.username. I think it works on form only not on user object.